scripts/article-views/update-article-views/utils.ts

```diff
@@ -10,7 +10,7 @@ export const hashString = (str: string) => {
 
 export const amplitudeFetchJSON = async (chart_id: string) => {
   const url = `https://reops-proxy.intern.nav.no/amplitude/100002016/api/3/chart/${chart_id}/query`;
-  console.log({ url });
+  console.info({ url });
```

Comment:
Not familiar with this script, but this looks a little bit like debug logging...? 🧐
I.e. should it be removed?